### PR TITLE
Fix test for ocl subscription module

### PR DESCRIPTION
**Pull request by kdaud**
The commit at https://github.com/openmrs/openmrs-contrib-qaframework/commit/61c6a324d1b2579cbf32bd619af10914a14e906f made ocl test to break as per the logs at https://github.com/openmrs/openmrs-contrib-qaframework/runs/5984585516?check_suite_focus=true

### CODE REVIEW

**Comment by sherrif10** [comment]
Add the description of these tickets, probably it is recommeded to include ticket reference if its there.

**Comment by kdaud** [comment]
Have done the needful though I am a little lazy to log a ticket for this change, the description have added is sufficient.

File: qaframework-bdd-tests/package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "refapp3Settings": "cypress run --spec src/test/resources/features/refapp-3.x/03-settings/settings.feature",
     "refapp3ClinicalVisit": "cypress run --spec src/test/resources/features/refapp-3.x/04-clinical-visit/clinical-visit.feature",
     "refapp3VitalsAndTriage": "cypress run --spec src/test/resources/features/refapp-3.x/05-vitals-and-triage/vitals-and-triage.feature",
-    "refappOclSubscriptionModule": "cypress run --spec src/test/resources/features/oclsubscriptionmodule/ocl-subscription-module.feature"
```
Comment (sherrif10) [inline qaframework-bdd-tests/package.json:47]: Seems you are changing from oclsubsription to ocl right  directory ?

**Review by sherrif10 — COMMENTED** [review]
LGTM

**Review by sherrif10 — APPROVED** [review]
LGTM

**Comment by sherrif10** [comment]
Thanks @kdaud

**Pull request merged**